server/src/libs/api-gateway.ts

```diff
@@ -0,0 +1,12 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyResult, Handler } from "aws-lambda"
+import type { FromSchema } from "json-schema-to-ts";
+
+type ValidatedAPIGatewayProxyEvent<S> = Omit<APIGatewayProxyEvent, 'body'> & { body: FromSchema<S> }
+export type ValidatedEventAPIGatewayProxyEvent<S> = Handler<ValidatedAPIGatewayProxyEvent<S>, APIGatewayProxyResult>
+
+export const formatJSONResponse = (response: Record<string, unknown>) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify(response)
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/73526360/221542090-a47a9340-6f30-4cd5-93e2-c58f408d4f52.png)
Probably should add headers.
this post can help:
![image](https://user-images.githubusercontent.com/73526360/221542395-12744fd5-fa86-4656-9339-27e83694ab60.png)

Reply:
Yes, I fixed cors' issues. Here is the cloud front link https://d2qop4n0v0xrf.cloudfront.net/